UnityProject/Assets/Scripts/UI/Objects/Cargo/GUI_Cargo.cs

```diff
@@ -49,20 +61,47 @@ public void OpenTab(NetPage pageToOpen)
 			var cargopage = pageToOpen.GetComponent<GUI_CargoPage>();
 			cargopage.OpenTab();
 			cargopage.UpdateTab();
-			DirectoryText.MasterSetValue(cargopage.DirectoryName);
+		}
+
+		public void OpenCategory(int category)
+		{
+			pageSupplies.cargoCategory = categories[category];
+			OpenTab(pageSupplies);
 		}
 
 		private void UpdateCreditsText()
 		{
 			if(CargoManager.Instance.CargoOffline)
 			{
-				СreditsText.MasterSetValue("OFFLINE");
+				СreditsText.SetValue("OFFLINE");
 				return;
 			}
-			СreditsText.MasterSetValue($"Budget: {CargoManager.Instance.Credits}");
+			СreditsText.SetValue($"Credits:\n{CargoManager.Instance.Credits}");
 			if (cargoConsole != null) { cargoConsole.PlayBudgetUpdateSound(); }
 		}
 
+		private void UpdateStatusText()
+		{
+			string[] statusText = new string[4] { "On-Route Station", "Docked at Station", "On-Route Centcomm", "Docked at Centcomm" };
+
+			if (CargoManager.Instance.CargoOffline)
+			{
+				StatusText.SetValue("OFFLINE");
+				return;
+			}
+			if(CargoManager.Instance.ShuttleStatus == ShuttleStatus.DockedStation)
+			{
+				raiseButtonText.SetValue("    Send");
```

Comment:
why the spaces

Reply:
It was mainly to save some pain trying to get the UI to line up visually, the text boxes conform to the button dimensions, but the buttons also have the icons, the space is so the text starts after the icon without having to mess with the stretchy stuff with the text boxes as that gets very frustrating.

Reply:
I have a feeling this isn't the best solution since it might not look correctly on different displays or whenever someone updates the UI again in the future.

Reply:
Aight, Ill go through and adjust it